exec/java-exec/src/main/java/org/apache/drill/exec/store/parquet/ParquetRecordWriter.java

```diff
@@ -380,14 +384,21 @@ public void endRecord() throws IOException {
 
       // since ParquetFileWriter will overwrite empty output file (append is not supported)
       // we need to re-apply file permission
-      parquetFileWriter = new ParquetFileWriter(conf, schema, path, ParquetFileWriter.Mode.OVERWRITE);
+      if (useConfiguredBlockSize) {
+        // Round up blockSize to multiple of 64K.
+        long writeBlockSize = ((long) ceil((double)blockSize/BLOCKSIZE_MULTIPLE)) * BLOCKSIZE_MULTIPLE;
```

Comment:
Underlying FS API expects blockSize specified during create to be multiple of 64K.  So, I rounded it up.  This will be greater than (to nearest 64K) or equal to what is used in checkBlockSizeReached.  Should I change checkBlockSizeReached to do the same ? I was trying to avoid any changes to existing behavior/code.